blog/_src/posts/2022-06-21-shallow-typed-racket.scrbl

```diff
@@ -0,0 +1,335 @@
+#lang scribble/manual
+
+Title: Shallow and Optional Typed Racket
+Date: 2022-06-21T23:43:38
+Tags: Typed Racket
+
+@(require
+   scribble/example
+   (only-in scribble/eval interaction-eval interaction/no-prompt)
+   (for-label typed/racket/base))
+
+@(define deep-eval (make-base-eval #:lang 'typed/racket))
+@(define shallow-eval (make-base-eval #:lang 'typed/racket/shallow))
+@(define optional-eval (make-base-eval #:lang 'typed/racket/optional))
+
+@(define (tech/reference . text)
+   (keyword-apply tech '(#:doc) '((lib "scribblings/reference/reference.scrbl")) text))
+
+@(define-syntax-rule (module-example #:eval ev #:label lbl lang-datum mod-code ... ex-code)
+   (list
+     (if lbl (list lbl) '())
+     (racketmod lang-datum mod-code ...)
+     (interaction-eval #:eval ev mod-code ...)
+     (interaction/no-prompt #:eval ev ex-code)))
+
+With the Racket 8.7 release, Typed Racket (TR) includes two languages that
```

Comment:
Do you truly mean 8.7 and not 8.6? (because we are currently releasing 8.6).

Reply:
Yes, this has not been merged yet.